app/code/Magento/Customer/Controller/AbstractAccount.php

```diff
@@ -9,9 +9,11 @@
 use Magento\Framework\App\Action\Action;
 
 /**
- * Class AbstractAccount
- * @package Magento\Customer\Controller
+ * AbstractAccount class is deprecated, in favour of Composition approach to build Controllers
+ *
  * @SuppressWarnings(PHPMD.NumberOfChildren)
+ * @deprecated 2.4.0
```

Comment:
```suggestion
 * @deprecated
```
2.4.0 is a marketing version. Module version will be added by automation tools.